src/test/java/Acceptance/AcceptanceTests.java

```diff
@@ -155,4 +155,64 @@ public void addLeague2ATest(){
         EntityManager.getInstance().removeLeagueByName("Premier League");
         assertFalse(EntityManager.getInstance().doesLeagueExists("Premier League"));
     }
+
+    /**
+     * 6.1.1.a
+     * User for Elisha Levy exists
+     */
+    @Test
+    public void addAsset1ATest() throws Exception
+    {
+        Team beitShean = new Team();
+
+        beitShean.setTeamName("Beit Shean");
+        SystemUser abcCreate = new SystemUser("abc1","abc12345","abc");
```

Comment:
From my experience, I think you should do a System Users' cleanup at the end each test. In the SystemUser constructor  you automatically add the user to EntityManager, so at the end of each test do "EntityManager.RemoveUser(user)" in order to prevent problems when different tests creating SUsers with the same usernames, and all of them are running together.